src/group_theory/nilpotent.lean

```diff
@@ -122,11 +122,11 @@ def upper_central_series (n : ℕ) : subgroup G := (upper_central_series_aux G n
 
 instance (n : ℕ) : normal (upper_central_series G n) := (upper_central_series_aux G n).2
 
-lemma upper_central_series_zero_def : upper_central_series G 0 = ⊥ := rfl
+@[simp] lemma upper_central_series_zero_def : upper_central_series G 0 = ⊥ := rfl
```

Comment:
I would remove the `_def` here and below. I think we usually only use `X_def` when `X` itself is already a declaration.